app/models/course/assessment/question/text_response_comprehension_solution.rb

```diff
@@ -32,4 +34,9 @@ def strip_whitespace_solution
   def strip_whitespace_solution_lemma
     solution_lemma.each(&:strip!)
   end
+
+  # add custom error message for `solution_lemma` instead of default :blank
```

Comment:
Cannot overwrite the existing `:blank` error message as it is stil required in `solution`.